Core/Core/Extensions/UITableViewHeaderFooterViewExtensions.swift

```diff
@@ -0,0 +1,61 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2020-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import UIKit
+
+@available(iOSApplicationExtension 13.0.0, *)
+extension UITableViewHeaderFooterView {
+    @objc dynamic var hasBorderSeparators: Bool {
```

Comment:
I don't like this sort of thing. I also don't understand why we've never needed this before. Is it because we are setting a background color on these, that covers up the cell's separators?

Reply:
Separator's aren't drawn for headers in a tableview, but they're present in Figma. We could instead implement the headers just as cells, but that loses the floating behavior (which may not be that important)